docs/building-apps.md

```diff
@@ -8,29 +8,34 @@ order: 4
     enact pack [options]
 
   Options
+    -o, --output      Specify an output directory
     --content-hash    Add a unique hash to output file names based on the content of an asset
+    -w, --watch       Rebuild on file changes
     -p, --production  Build in production mode
     -i, --isomorphic  Use isomorphic code layout
                       (includes prerendering)
-    -w, --watch       Rebuild on file changes
     -l, --locales     Locales for isomorphic mode; one of:
-            <comma-separated-values> Locale list
-            <JSON-filepath> - Read locales from JSON file
+            \<comma-separated-values\> Locale list
+            \<JSON-filepath\> - Read locales from JSON file
```

Comment:
< > phrases are still not visible on docs.

Reply:
ok